internal/tiltfile/extension/store.go

```diff
@@ -41,7 +39,7 @@ func NewLocalStore(baseDir string) *LocalStore {
 }
 
 func (s *LocalStore) ModulePath(ctx context.Context, moduleName string) (string, error) {
-	tiltfilePath := filepath.Join(s.baseDir, moduleName, tiltfile.FileName)
+	tiltfilePath := filepath.Join(s.baseDir, moduleName, "Tiltfile")
```

Comment:
Had to use the bare string here to prevent a import cycles.

Reply:
probably doesn't matter much, but maybe make an ExtensionTiltfileName const in this package?